.ci/openvino-onnx/Dockerfile

```diff
@@ -76,9 +76,4 @@ RUN make -j $(nproc) install
 
 # Run tests via tox
 WORKDIR /openvino/ngraph/python
-ENV NGRAPH_CPP_BUILD_PATH=/openvino/dist
```

Comment:
Please don't need to remove these env vars from here. I think you can override the ones which are not default in `docker exec` but these are a useful form of documentation.